### ISSUE TITLE
Quest: Diabolical Plans

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#81]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Ashenvale / Felfire Hill / Item: Diabolical Plans / Quest: Diabolical Plans
Link: https://www.wowhead.com/wotlk/quest=9535/diabolical-plans

Description: the voice-over suddenly silences at the end of the playback.

Progress: no playback.

Completion: playback ok.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_TBC_v1.4.
